inst/extdata/database/flow_types.csv

```diff
@@ -1,27 +1,358 @@
+buoh,density_std,3065 g/gal,GREET2008,"not explicitly given if NTP or STP, assumed that value does not change muchh in this temperature range"
+ng,name,Natural gas,,
+ng,default_unit,MWh;LHV,,
+ng,energycontent_LHV,21.626 Btu/kg,GREET2008,calculated from values Energy/volume and density given in the source
+ng,energycontent_HHV,23.958 Btu/kg ,GREET2008,calculated from values Energy/volume and density given in the source
+ng,density_norm,22 g/ft^3,GREET2008,"not explicitly given if NTP or STP, assumed that value does not change muchh in this temperature range"
+ng,density_std,22 g/ft^3,GREET2008,"not explicitly given if NTP or STP, assumed that value does not change muchh in this temperature range"
+coal,name,Coal,,
+coal,default_unit,MWh;LHV,,
+coal,energycontent_LHV,22732 MJ/kg,Boundy2011,
+coal,energycontent_HHV,23968 MJ/kg,Boundy2011,
+coal,density_norm,,,
+coal,density_std,,,
+co2,name,Purified CO2,,
+co2,default_unit,t,,
+co2,energycontent_LHV,,,
+co2,energycontent_HHV,,,
+co2,density_norm,1.8393 kg/m³,Lemmon2023,
+co2,density_std,1.9768 kg/m³,Lemmon2023,
+ironore,name,Iron ore,,
+ironore,default_unit,t,,
+ironore,energycontent_LHV,,,
+ironore,energycontent_HHV,,,
+ironore,density_norm,,,
+ironore,density_std,,,
+dri,name,Directly reduced iron,,
+dri,default_unit,t,,
+dri,energycontent_LHV,,,
+dri,energycontent_HHV,,,
+dri,density_norm,,,
+dri,density_std,,,
+alloys,name,Alloys,,
+alloys,default_unit,t,,
+alloys,energycontent_LHV,,,
+alloys,energycontent_HHV,,,
+alloys,density_norm,,,
+alloys,density_std,,,
+lime,name,Lime,,
+lime,default_unit,t,,
+lime,energycontent_LHV,,,
+lime,energycontent_HHV,,,
+lime,density_norm,,,
+lime,density_std,,,
+steelliq,name,Liquid steel,,
+steelliq,default_unit,t,,
+steelliq,energycontent_LHV,,,
+steelliq,energycontent_HHV,,,
+steelliq,density_norm,,,
+steelliq,density_std,,,
+steelslab,name,Steel slab,,
+steelslab,default_unit,t,,
+steelslab,energycontent_LHV,,,
+steelslab,energycontent_HHV,,,
+steelslab,density_norm,,,
+steelslab,density_std,,,
+steelhrc,name,Hot-rolled coil,,
+steelhrc,default_unit,t,,
+steelhrc,energycontent_LHV,,,
+steelhrc,energycontent_HHV,,,
+steelhrc,density_norm,,,
+steelhrc,density_std,,,
+steelscrap,name,Steel scrap,,
+steelscrap,default_unit,t,,
+steelscrap,energycontent_LHV,,,
+steelscrap,energycontent_HHV,,,
+steelscrap,density_norm,,,
+steelscrap,density_std,,,
+hvcs,name,Higher-value chemicals,,
+hvcs,default_unit,t,,
+hvcs,energycontent_LHV,,,
+hvcs,energycontent_HHV,,,
+hvcs,density_norm,,,
+hvcs,density_std,,,
+urea,name,Urea,,
+urea,default_unit,t,,
+urea,energycontent_LHV,,,
+urea,energycontent_HHV,,,
+urea,density_norm,1.34 g/cm³,,
+urea,density_std,1.34 g/cm³,,
+water,name,Water,,
+water,default_unit,t,,
+water,energycontent_LHV,,,
+water,energycontent_HHV,,,
+water,density_norm,998.21  kg/m³,Lemmon2023,
+water,density_std,999.84 kg/l,Lemmon2023,
```

Comment:
I guess the unit here should be `kg/m³` too?

Reply:
yes it should